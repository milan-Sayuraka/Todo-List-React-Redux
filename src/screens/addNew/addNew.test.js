import renderer from 'react-test-renderer';
import React, { useState } from 'react';
import { useDispatch } from 'react-redux';
import { useNavigate } from "react-router-dom";
import AddNew from './addNew';

jest.mock('react-redux');
jest.mock("react-router-dom");

const renderTree = tree => renderer.create(tree);
describe('<AddNew>', () => {
  it('should render component', () => {
    expect(renderTree(<AddNew 
    />).toJSON()).toMatchSnapshot();
  });
  
});