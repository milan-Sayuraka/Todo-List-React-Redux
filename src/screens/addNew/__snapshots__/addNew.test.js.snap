// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddNew> should render component 1`] = `
<section
  id="section-todo"
>
  <h3
    className="center-align white-text blue"
  >
    Add New Todo
  </h3>
  <div
    className="row"
  >
    <p
      className="red-text err-msg col s12 center-align"
    />
    <div
      className="input-field col s10"
    >
      <input
        id="todo-input"
        onChange={[Function]}
        placeholder="Type todo here"
        type="text"
        value=""
      />
      <label
        className="active"
        htmlFor="todo-input"
      >
        New Todo
      </label>
    </div>
    <button
      className="btn col s2 blue"
      onClick={[Function]}
    >
      Add
    </button>
  </div>
</section>
`;
